### PR TITLE
PHP8.4 Support Added

### PR DESCRIPTION
**Pull request by cod43156**
|    Q          |   A
|-------------- | ------
| Documentation | yes/no
| Bugfix        | yes/no
| BC Break      | yes/no
| New Feature   | yes/no
| RFC           | yes/no
| QA            | yes/no

### CODE REVIEW

File: docs/book/v3/naming-strategy/composite-naming-strategy.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 `Laminas\Hydrator\NamingStrategy\CompositeNamingStrategy` allows you to specify which naming
 strategy should be used for each key encountered during hydration or extraction.
 
+<!-- markdownlint-disable-next-line single-h1 single-title -->
```
Comment (froschdesign) [inline docs/book/v3/naming-strategy/composite-naming-strategy.md]: Not required here, as the associated heading is in the wrong order.

Comment (cod43156) [reply docs/book/v3/naming-strategy/composite-naming-strategy.md]: @froschdesign https://docs.laminas.dev/laminas-hydrator/v3/naming-strategy/composite-naming-strategy/ I see this doc has both CompositeNamingStrategy and Basic Usage as H1. Still should I make Basic Usage as H2?

Comment (froschdesign) [reply docs/book/v3/naming-strategy/composite-naming-strategy.md]: It must be `h2`.

Comment (cod43156) [reply docs/book/v3/naming-strategy/composite-naming-strategy.md]: Updated

**Comment by gsteel** [comment]
Thanks @cod43156 - this has been handled via #124

**Pull request closed without merging**